src/components/AlbumListItem.tsx

```diff
@@ -36,17 +39,36 @@ export class AlbumListItem extends React.Component<Props, State> {
   }
 
   render() {
+    const songList = this.state.showSongList || this.props.isSearching
+    ? (
+      <SongList 
+        album={this.props.album}
+        directSongMatches={this.props.directSongMatches}
+        isExpanded={true}
+        isSearching={this.props.isSearching}
+        songs={this.props.songs}
+        onSongClick={this.props.onSongClick}
+      />
+    )
+    : null;
     return (
       <li key={`album-${this.props.album.id}`}>
-        <div onClick={() => this.setState({showSongList: !this.state.showSongList})}>
+        <div onClick={this.onClick.bind(this)}>

```

Comment:
extraneous `div` is unnecessary. click should be an event bubbled up from `AlbumListItemHeader`